vindemitor/core/post_processor.py

```diff
@@ -128,3 +125,19 @@ def _mux_media(self, title: Title_T, tracks: Tracks, progress_callback) -> Path:
             video_track.delete()
 
         return muxed_path
+
+    def _tag_file(self, path, tag, title: Title_T):
+        xml_lines = ['<?xml version="1.0" encoding="UTF-8"?>', "<Tags>", "  <Tag>", "    <Targets/>"]
+        xml_lines.append(f"    <Simple><Name>Description</Name><String>{title.description}</String></Simple>")
```

Comment:
This will raise `AttributeError` when `Song` is passed. Either guard it or add `description` attribute to `Song` class, whichever made more sense.